internal/shell.go

```diff
@@ -63,6 +67,9 @@ func shJSONPipe[T any](prog string, args shArgs, stdIn string, eventPipe chan<-
 	err = cmd.Wait()
 
 	if err != nil {
+		// print out the stdout back to stdout so we can debug
+		fmt.Fprintln(os.Stderr, stdErr.String())
```

Comment:
not related to the current PR but helps with debugging

Reply:
also if for example the go code cannot be built, it's printed here

Reply:
Yes! This is bug I had noticed. Nice!